aws/src/test/java/org/apache/iceberg/aws/TestAwsClientFactories.java

```diff
@@ -57,6 +57,7 @@ public void testLoadCustom() {
   }
 
   @Test
+  @Deprecated
```

Comment:
no need to mark test as deprecated, we should update tests to reflect the latest change